azavea/iow-boundary-tool
# Sidebar prototype

## Overview

The PR adds the UI elements to the draw page sidebar and some basically visibility toggle functionality. 

Closes #23 

### Demo

<img width="318" alt="image" src="https://user-images.githubusercontent.com/8356789/184421836-c64f0f3c-19c7-45b9-ab50-a2bf0ffa9b26.png">

## Testing Instructions

- Go to http://localhost:4545/draw
- Sidebar should be on right side of page
- Check that UI items match Figma diagram

## Checklist
